src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -95,7 +97,7 @@ function convertDateToHHMM(date) {
 function selectDefaultTimeOptions() {
     const now = new Date();
 
-    const currentDate = convertDateToDDMMYYYY(now);
+    const currentDate = convertDateTodddDDMMMYYYY(now);
     const hours = convertDateToHHMM(now).substring(0, 2);
     const currentTime = parseInt(hours, 10) + 1;
     const timeZone = -now.getTimezoneOffset() / 60;
```

Comment:
This entire chunk is unnecessary, e.g.:
```patch
 function selectDefaultTimeOptions() {
     const now = new Date();
 
-    const currentDate = convertDateTodddDDMMMYYYY(now);
-    const hours = convertDateToHHMM(now).substring(0, 2);
-    const currentTime = parseInt(hours, 10) + 1;
-    const timeZone = -now.getTimezoneOffset() / 60;
-
     if (!isTimeZoneIntialized()) {
-        $(`#${ParamsNames.FEEDBACK_SESSION_STARTDATE}`).val(currentDate);
-        $(`#${ParamsNames.FEEDBACK_SESSION_STARTTIME}`).val(currentTime);
-        $(`#${ParamsNames.FEEDBACK_SESSION_TIMEZONE}`).val(timeZone);
+        $(`#${ParamsNames.FEEDBACK_SESSION_STARTDATE}`).datepicker('setDate', now);
+        $(`#${ParamsNames.FEEDBACK_SESSION_STARTTIME}`).val(now.getHours());
+        $(`#${ParamsNames.FEEDBACK_SESSION_TIMEZONE}`).val(-now.getTimezoneOffset() / 60);
     }
 
     const uninitializedTimeZone = $(`#timezone > option[value='${TIMEZONE_SELECT_UNINITIALISED}']`);
```

Reply:
@whipermr5 Adding this line  ```$(`#${ParamsNames.FEEDBACK_SESSION_STARTDATE}`).datepicker('setDate', now);``` results in wrongly placed datepicker.
![datep](https://user-images.githubusercontent.com/24381442/36852676-54e85eb6-1d92-11e8-862a-4e00a0bc3325.png)
I tried 
- hiding it ```$(`#${ParamsNames.FEEDBACK_SESSION_STARTDATE}`).datepicker('hide');```, 
- specifying container of datepicker in datepicker.js, and some other stackoverflow solutions but it didn't work. 
Could you please suggest something?